rudder-server-root/SOURCES/Makefile

```diff
@@ -29,6 +36,23 @@ localdepends: ./rudder-sources
 	tar -xjf rudder-sources.tar.bz2
 	mv rudder-sources-*/ rudder-sources/
 
+build:
+	true
+
+install:
+	mkdir -p $(DESTDIR)/etc/init.d/
+
+	# Init files
+ifneq (true,$(NO_INIT))
+	$(INSTALL) -m 755 rudder-server.init $(DESTDIR)/etc/init.d/rudder-server
```

Comment:
There is no reference to this file in the specfile/debianrules how does it suddenly appear here